retrieval/requirements.txt

```diff
@@ -0,0 +1,16 @@
+packaging
+dataclasses
+fire
+numpy
+pandas
+pyarrow
+tqdm
+fsspec
+embedding_reader
+huggingface_hub
+regex
+sacremoses
+importlib_metadata
+tokenizers
```

Comment:
Apparently 'rich' is also a needed package

Reply:
Yes, there are 3 unlisted dependencies. Let me update and push an additional commit to this PR.